src/test/java/org/broadinstitute/hellbender/tools/walkers/validation/EvaluateGenotypingPerfromanceUnitTests.java

```diff
@@ -0,0 +1,178 @@
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+
+public class EvaluateGenotypingPerfromanceUnitTests extends GATKBaseTest {
+
+    @Test
+    void testPearsonCorrelationAggregator() {
+        final Random rand = new Random(234);
+        final int nEntries = 1000;
+        final double[] x = new double[nEntries];
+        final double[] y = new double[nEntries];
+
+        final EvaluateGenotypingPerformance.PearsonCorrelationAggregator aggregator = new EvaluateGenotypingPerformance.PearsonCorrelationAggregator();
+
+        for (int i = 0; i < nEntries; i++) {
+            x[i] = rand.nextDouble();
+            y[i] = rand.nextDouble();
+
+            aggregator.addEntry(x[i], y[i]);
+        }
+
+        final PearsonsCorrelation pearsonsCorrelation = new PearsonsCorrelation();
+        Assert.assertEquals(aggregator.getCorrelation(), pearsonsCorrelation.correlation(x, y), 1e-10);
+
+    }
+
+    @DataProvider(name = "getConcordanceStateDataProvider")
+    Object[][] getConcordanceStateDataProvider() {
+        return new Object[][] {
+                {Arrays.asList(Allele.ALT_A, Allele.REF_G), Arrays.asList(Allele.REF_G, Allele.ALT_A), false, ConcordanceState.TRUE_POSITIVE},
+                {Arrays.asList(Allele.ALT_A, Allele.REF_G), Arrays.asList(Allele.REF_G, Allele.ALT_A), true, ConcordanceState.FILTERED_FALSE_NEGATIVE},
+                {Arrays.asList(Allele.ALT_A, Allele.ALT_A), Arrays.asList(Allele.REF_G, Allele.ALT_A), false, ConcordanceState.FALSE_POSITIVE},
+                {Arrays.asList(Allele.ALT_A, Allele.ALT_A), Arrays.asList(Allele.REF_G, Allele.ALT_A), true, ConcordanceState.FILTERED_FALSE_NEGATIVE},
+                {Arrays.asList(Allele.ALT_A, Allele.ALT_A), Arrays.asList(Allele.ALT_G, Allele.ALT_A), false, ConcordanceState.FALSE_POSITIVE},
+                {Arrays.asList(Allele.ALT_A, Allele.ALT_A), Arrays.asList(Allele.ALT_A, Allele.ALT_A), false, ConcordanceState.TRUE_POSITIVE},
+                {Arrays.asList(Allele.REF_A, Allele.REF_A), Arrays.asList(Allele.REF_A, Allele.REF_A), false, ConcordanceState.TRUE_NEGATIVE},
+                {Arrays.asList(Allele.REF_A, Allele.REF_A), Arrays.asList(Allele.REF_A, Allele.REF_A), true, ConcordanceState.TRUE_NEGATIVE},
+                {Arrays.asList(Allele.REF_A, Allele.REF_A), Arrays.asList(Allele.REF_A, Allele.ALT_C), false, ConcordanceState.FALSE_POSITIVE},
+                {Arrays.asList(Allele.REF_A, Allele.REF_A), Arrays.asList(Allele.REF_A, Allele.ALT_C), true, ConcordanceState.FILTERED_TRUE_NEGATIVE},
+
+                //differing ploidy, generally from different treatment of chromosome X in males
+
+                {Arrays.asList(Allele.REF_A, Allele.REF_A), Arrays.asList(Allele.REF_A), false, ConcordanceState.TRUE_NEGATIVE},
+                {Arrays.asList(Allele.ALT_A, Allele.ALT_A), Arrays.asList(Allele.REF_A), false, ConcordanceState.FALSE_NEGATIVE},
```

Comment:
It is my understanding that this is not possible; might want ot create more realistic test scenarios (i.e. ref != alt)